src/main/java/org/opentripplanner/middleware/triptracker/TravelerLocator.java

```diff
@@ -68,6 +85,15 @@ private static boolean hasRequiredWalkLeg(TravelerPosition travelerPosition) {
             travelerPosition.expectedLeg.mode.equalsIgnoreCase("walk");
     }
 
+    /**
+     * Has required walk leg.
```

Comment:
Has required transit leg??

Reply:
Yeah, dealt with the remaining copy pasta in https://github.com/ibi-group/otp-middleware/pull/238/commits/7bd7ff94e83f5de5f7dd281a07ed8aaee073e130.